frontend/src/features/videos/VideoList.tsx

```diff
@@ -26,37 +23,17 @@ function VideoList({
   const classes = useStyles();
   const { isLoggedIn } = useLoginState();
 
-  const AddToRateLaterList = ({ videoId }: { videoId: string }) => {
-    const video_id = videoId;
-    return (
-      <div>
-        {isLoggedIn && (
-          <IconButton
-            size="medium"
-            color="secondary"
-            onClick={async () => {
-              await UsersService.usersMeVideoRateLaterCreate({
-                video: { video_id } as Video,
-              });
-            }}
-          >
-            <AddIcon />
-          </IconButton>
-        )}
-      </div>
-    );
-  };
+  const actions = isLoggedIn
+    ? [CompareNowAction, AddToRateLaterList]
+    : [CompareNowAction];
 
   return (
     <div>
       {videos.results?.length ? (
         videos.results.map((video: VideoSerializerWithCriteria) => (
           <Grid container className={classes.card} key={video.video_id}>
             <Grid item xs={12}>
-              <VideoCard
-                video={video}
-                actions={[CompareNowAction, AddToRateLaterList]}
-              />
+              <VideoCard video={video} actions={actions} />
```

Comment:
I don't fully understand this change. If the user is not logged in, he can not add videos to his rate later list, neither compare it

Reply:
I did not change from the previous behaviour. But you are right. I'll remove the link to comparison

Reply:
Indeed it was my mistake at the beginning